# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/LocalAccessible.cpp
# Commit: 6d0f4821ddcf
# Full Hash: 6d0f4821ddcfca1b59e4db3befc3ec9c12335891
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-03-23 09:49:32
# Regressor Bug: 1735970
# File Overlap Count: 1
# Description:
#   Bug 1735970 part 3: Add TableAccessibleBase and TableCellAccessibleBase. r=morgan
#   
#   These are needed to support RemoteAccessible tables.
#   Stuff specific to LocalAccessible is still in TableAccessible and TableCellAccessible, which now inherit from the new Base classes.
#   
# ==============================================================================

diff -r d82660fb5408 -r 6d0f4821ddcf accessible/generic/LocalAccessible.cpp
--- a/accessible/generic/LocalAccessible.cpp	Wed Mar 23 04:00:25 2022 +0000
+++ b/accessible/generic/LocalAccessible.cpp	Wed Mar 23 04:00:26 2022 +0000
@@ -3543,3 +3543,9 @@
 
   aValue.Append(mKey);
 }
+
+TableAccessibleBase* LocalAccessible::AsTableBase() { return AsTable(); }
+
+TableCellAccessibleBase* LocalAccessible::AsTableCellBase() {
+  return AsTableCell();
+}